helix-term/src/compositor.rs

```diff
@@ -248,6 +248,17 @@ impl Compositor {
             .find(|component| component.id() == Some(id))
             .and_then(|component| component.as_any_mut().downcast_mut())
     }
+
+    pub fn claim_term(&mut self) -> Result<(), termwiz::Error> {
+        self.terminal.terminal().enter_alternate_screen()?;
+        self.terminal.terminal().set_raw_mode()
+    }
+
+    pub(crate) fn clear(&mut self) -> Result<(), termwiz::Error> {
+        let inner_term = self.terminal.terminal();
+        inner_term.exit_alternate_screen()?;
+        inner_term.set_cooked_mode()
+    }
```

Comment:
I think it's kinda confusing that one is called `claim_term` and the other `clear`.

Reply:
The cursor methods should be similarly renamed from `save_cursor`/`load_cursor`